Sprint-3/4-stretch-investigate/password-validator.test.js

```diff
@@ -14,13 +14,30 @@ To be valid, a password must:
 
 You must breakdown this problem in order to solve it. Find one test case first and get that working
 */
-const isValidPassword = require("./password-validator");
+//const isValidPassword = require("./password-validator");
+
+
+function passwordValidator(password) {
```

Comment:
No!  You should be using require to get the existing function in from password-validator.js.  We want to separate our main code (that we want to include in our applications) from our test code (that we don't), but we definitely want to duplicate the code.  Have a search for how require works in javascript